backend/app/api/routes/model_management.py

```
@@ -0,0 +1,263 @@
+    _current_user: User = Depends(get_current_user),
+):
+    """
+    动态添加新的模型配置
+    """
+    try:
+        # 准备 LiteLLM 模型配置格式
+        litellm_config = {
+            "model_name": model_config.model_name,
+            "litellm_params": model_config.litellm_params
+        }
+        
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)